src/components/SingleTermView/MergePanel/MergePanel.tsx

```diff
@@ -0,0 +1,86 @@
+
+const styles = {
+  title: {
+    fontSize: "1.125rem",
+    color: gray800,
+    fontWeight: 600,
+    lineHeight: "1.75rem"
+  },
+  chip: {
+    border: `1px solid ${gray300}`,
+    background: white,
+    fontSize: "0.875rem"
+  }
+}
+
+const keyToTitleMap = {
+  versionInfo: "Version",
+  hasIlxPreferredId: "Preferred ID",
+  description: "Description",
+  type: "Type",
+  owlEquivalent: "OWL equivalent",
+  submittedBy: "Originally submitted by",
+  lastModifiedBy: "Last modified by",
+  lastModifyTimestamp: "Last modify timestamp",
+  predicates: "Predicates"
+}
+
+const orderedKeys = ["synonym", "existingID", "hasIlxPreferredId", "description", "type", "versionInfo", "owlEquivalent", "submittedBy", "lastModifiedBy", "lastModifyTimestamp", "predicates"]
+
+
+const MergePanel = ({ data, compareData, status }) => {
+
+  const getTitle = (key: string): string => {
+    return keyToTitleMap[key] || key.replace(/([A-Z])/g, " $1").replace(/^./, (str) => str.toUpperCase())
+  }
+
+  return (
+    <Grid container xs={12} lg={6} sx={{ borderRight: `1px solid ${gray200}`, backgroundColor: gray25, overflowY: 'auto' }}>
+      <Box display="flex" justifyContent="space-between" sx={{ padding: "1.5rem", borderBottom: `1px solid ${gray200}`, background: "#fff", maxHeight: "4.75rem" }} width={1}>
+        <Typography sx={styles.title}>{data?.label}</Typography>
+        <Chip label={"Curated"} sx={styles.chip} />
+      </Box>
+      <Grid container p={3} spacing={5.5}>
+        {Object.keys(data).sort((a, b) => orderedKeys.indexOf(a) - orderedKeys.indexOf(b)).map(property => {
+          const component = mapping[property]
+          switch (component) {
+            case "ChipSynonymChanges":
+              return <Grid item lg={12}>
+                  <ChipChanges data={data[property]} compareData={compareData[property]} status={status} type="synonym" />
+                </Grid>
+            case "ChipExistingIDChanges":
+              return <Grid item lg={6} xs={12}>
+                  <ChipChanges data={data[property]} compareData={compareData[property]} status={status} type="existingID" />
+                </Grid>
+            case "LabelChanges":
+                return <Grid item lg={6} xs={2} sm={4} md={4}>
+                  <LabelChanges title={getTitle(property)} data={data[property]} compareData={compareData[property]} status={status} />
+                </Grid>
+            case "TextChanges":
+                return <Grid item lg={12}>
+                  <TextChanges title={getTitle(property)} data={data[property]} compareData={compareData[property]} status={status} />
+                </Grid>
+            case "TableChanges":
+                return <Grid item lg={12}>
+                  <PredicateChanges title={getTitle(property)} data={data[property]} compareData={compareData[property]} status={status} />
+                </Grid>
+            default:
+                return <></>
```

Comment:
I think we agreed that in case there is another property we would've checked for the type, if string or a number display it using a component (LabelChanges I guess), otherwise if it's a list use the ChipChanges component. Thanks

Reply:
And if it's something else just do a console.error to say that such property name is not supported since it's a complex structure/object.